projects/fmcjesdadc1/common/fmcjesdadc1_bd.tcl

```diff
@@ -129,3 +137,20 @@ ad_connect  $sys_dma_resetn axi_ad9250_dma/m_dest_axi_aresetn
 ad_cpu_interrupt ps-11 mb-14 axi_ad9250_jesd/irq
 ad_cpu_interrupt ps-13 mb-13 axi_ad9250_dma/irq
 
+# Create dummy outputs for unused Rx lanes
+for {set i $RX_NUM_OF_LANES} {$i < 4} {incr i} {
+    create_bd_port -dir I rx_data_${i}_n
+    create_bd_port -dir I rx_data_${i}_p
+}
+
+# Connect unused input pins in util_fmcjesdadc1_xcvr
+if {$RX_NUM_OF_LANES < 4} {
```

Comment:
This  comparison should be changed to $RX_NUM_OF_LANES == 2

Reply:
Changed the condition in the if-clause to $RX_NUM_OF_LANES == 2.